### DISCUSSION TITLE
Features pre-processing approach

### DISCUSSION THREAD

**Discussion by lamres** [autogluon/autogluon#1661] [Q&A] [answered]
Hi.
I assume that one algorithm works better with source features, the second one works better with min-max scaling, the third one works better when data is normalized by z-score, and so on.

Which approach is implemented for features pre-processing on tabular data? Suppose if I use default parameters for FASTAI, RF, and CAT algos.

Thanks!

**Comment by Innixma** [comment, marked as answer]
You can read [this tutorial](https://auto.gluon.ai/stable/tutorials/tabular_prediction/tabular-feature-engineering.html) for a basic overview of the global preprocessing pipeline. However, as you said each model can work better with different preprocessing. Thus, we have custom preprocessing for each ML model that is done on-top of the global preprocessing. This gets quite in-depth so if you are interested in how the model specific preprocessing works, I'd recommend reading the source code for the models listed [here](https://auto.gluon.ai/stable/api/autogluon.tabular.models.html).
